doc/modules/index.rst

```diff
@@ -10,6 +10,7 @@ This pages contains a list of available features in the library.
 
    mapper.rst
    homology.rst
+   local_homology.rst
```

Comment:
@jacobbamberger please also add the actual local_homology.rst file.

I recall we did add it to your local repo: we may ave forgotten to add it and commit it. Please do so.